Tests/CommandsTests/BuildCommandTests.swift

```diff
@@ -859,6 +859,12 @@ class BuildCommandSwiftBuildTests: BuildCommandTestCases {
     }
 
     override func testParseableInterfaces() async throws {
+        #if os(Linux)
+        if FileManager.default.contents(atPath: "/etc/system-release")
+                .map { String(decoding: $0, as: UTF8.self) == "Amazon Linux release 2 (Karoo)\n" } ?? false {
+            throw XCTSkip("https://github.com/swiftlang/swift-package-manager/issues/8545: Test currently fails on Amazon Linux 2")
```

Comment:
Note, this should be referencing https://github.com/swiftlang/swift-package-manager/issues/8560, not https://github.com/swiftlang/swift-package-manager/issues/8545.

#8560 is still open and tracks the actual fix, #8545 was just tracking the test failing because it wasn't properly being skipped.

Reply:
The GitHub issue reference is updated in https://github.com/swiftlang/swift-package-manager/pull/8569